### PR TITLE
Add dicebot

### PR DESCRIPTION
**Pull request by jrcasso**
### The basic pattern

We keep a list of functions to call whenever someone creates a message. We'll just dumbly pass the message to the functions in the list, and expect that they'll do the right thing with the message. If they return something, we'll go ahead and print that as a message to the user. If it's doesn't return anything (maybe something in the future will just operate entirely in the backend, and that's okay), so just move along. We'll add these functions to `handlers`, and then just loop through calling them in sequence. Now we can have many message processors!

The actual dice rolling code is on line 59.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,18 +10,56 @@
...
 async def on_message(message):
-    for word in message.content.lower().split():
-        if word.replace(',', '') in WHITELIST:
-            text_channels = discord.utils.get(client.get_all_channels(), guild__name=GUILD, name='general')
-            await text_channels.send(content="What do you want?")
+    text_channels = discord.utils.get(client.get_all_channels(), guild__name=GUILD, name='experiments')
+    # Every time a message is created, a number of handlers will receive the
+    # message, processing the message like an assembly line.
+    handlers = [
+        check_dice_roll
```
Comment (jrcasso) [inline main.py:33]: We can add another handler by creating a handler function (see line 46 - it takes one `message` arg) and adding it to this list.

File: main.py [hunk 2]
```diff
@@ -8,18 +10,56 @@
 client = discord.Client()
 
 
+class BrightException(Exception):
+    pass
+
+
+class Brightbot:
+    def __init__(self, text_channels):
+        self.text_channels = text_channels
```
Comment (jrcasso) [inline main.py:19]: I still need to implement this. I can rip it out for now if needed.

Comment (scmgustafson) [reply main.py:19]: implement the text_channels variable def or the function? sorry

Comment (jrcasso) [reply main.py:19]: Like, I created the class, but I never created an instance of it. It's just dead code; it's being compiled, but it's not currently being used. I'd have to call `bot = Brightenbot("general")` to use it.

can remove it

**Comment by jrcasso** [comment]
Closing (stale). Re-open if needed.

**Pull request closed without merging**